src/tinymce.js

```diff
@@ -54,29 +54,46 @@ angular.module('ui.tinymce', [])
         expression = {};
 
         angular.extend(expression, scope.$eval(attrs.uiTinymce));
+        tinymceWatchers.push(scope.$on('tinymce:init', function (ev, editorInstance) {
```

Comment:
What does moving these actions into event watchers help? Are they not just doing the same thing as when using the edit event handlers down below?

What does the extra event step give us?